_templates/component/new/Component.tsx.t

```diff
@@ -5,22 +5,19 @@ unless_exists: true
 <% PascalName = h.changeCase.pascal(name) %>
 import React from 'react';
 import classNames from 'classnames';
-import { ClassValue } from 'classnames/types';
 
 import styles from './<%= PascalName %>.module.css';
 
 interface <%= PascalName %>Props {
-  className?: ClassValue;
+  className?: string;
```

Comment:
I wanted to do this for a while, but my lazy justification for doing it now is that `ClassValue` is no longer available from the given import statement. I'm sure I could've found an alternative import, but it didn't seem worth it.

We were never intentionally exposing all the config options of `classNames` to consumers. They can pass plain old string class names with spaces between them, and that should be enough.

This also cleans up a handful of TS edge cases.